xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -115,24 +115,14 @@ public ConfigurationParser(String xml, URL... sources) throws IOException, SAXEx
     factory.setXIncludeAware(true);
     factory.setSchema(XSD_SCHEMA_FACTORY.newSchema(schemaSources.toArray(new Source[schemaSources.size()])));
 
-    final DocumentBuilder domBuilder;
-    try {
-      domBuilder = factory.newDocumentBuilder();
-    } catch (ParserConfigurationException e) {
-      throw new AssertionError(e);
-    }
+    DocumentBuilder domBuilder = factory.newDocumentBuilder();
```

Comment:
I thought assertions were for things that normally shouldn't occur. Messing up you XML is normal.